kasa/aestransport.py

```diff
@@ -233,7 +213,17 @@ async def perform_handshake(self):
 
         _LOGGER.debug(f"Request {request_body}")
 
-        status_code, resp_dict = await self.client_post(url, json=request_body)
+        cookies_dict = (
+            {self.SESSION_COOKIE_NAME: self._session_cookie}
+            if self._session_cookie
```

Comment:
Maybe store the cookies dict as a instance variable when session cookie is set so we don’t have to create the dict every time

Reply:
Done